java/java.completion/test/unit/src/org/netbeans/modules/java/completion/JavaCompletionTaskBasicTest.java

```diff
@@ -1298,4 +1298,9 @@ public void testAfterTypingMethodInvocationInMethodBody() throws Exception {
     public void testAfteMethodInvocationInMethodBody() throws Exception {
         performTest("SimpleMethodBody", 941, null, "methodBodyContentAndLocalVar.pass");
     }
+    public void testPropertyMethods() throws Exception {
+        performTest("ListPropertyMethods", 1170, null, "ListPropertyBool.pass");
+        performTest("ListPropertyMethods", 1199, null, "ListPropertyIntField.pass");
+        performTest("ListPropertyMethods", 1212, null, "ListPropertyIntField.pass");
+    }
```

Comment:
it might be required to create a test method per performTest call, since there is typically setup and teardown work involved which happens around each test method.

Reply:
Ah, ok